NEWS.md

```diff
@@ -1,4 +1,5 @@
 # Version 1.0.9000.9058
+* Renamed `SIM` argument of `nextBest` methods to `in_sim`.
```

Comment:
also mention `NEXT` replacement?

Reply:
actually `NEXT` is just internal variable, not user-facing.